src/main/java/sirius/biz/process/ProcessContext.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.process;
+
+import sirius.kernel.async.TaskContextAdapter;
+
+public interface ProcessContext extends TaskContextAdapter {
```

Comment:
do we really need this one as an interface? 
are there several different implementations planed?